charts/beekeeper/templates/clusterrole.yaml

```diff
@@ -43,8 +43,11 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   verbs: ["get", "create", "list", "watch"]
-- apiGroups: ["apps"]
-  resources: ["statefulsets"]
+- apiGroups: ["networking.k8s.io"]
+  resources: ["ingresses","ingressroutes"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["traefik.containo.us"]
+  resources: ["ingresses","ingressroutes"]
```

Comment:
Traefik is used only in the localcluster in CI? We are not installing beekeeper there like a job?